TenancyInformationApi.Tests/V1/Controllers/TenancyInformationControllerTests.cs

```diff
@@ -0,0 +1,55 @@
+using AutoFixture;
+using TenancyInformationApi.V1.Controllers;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using TenancyInformationApi.V1.UseCase.Interfaces;
+using Moq;
+using TenancyInformationApi.V1.Boundary.Response;
+
+namespace TenancyInformationApi.Tests.V1.Controllers
+{
+    [TestFixture]
+    public class TenancyInformationControllerTests
+    {
+        private TenancyInformationController _classUnderTest;
+        private Mock<IGetTenancyByIdUseCase> _getByIdMock;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _getByIdMock = new Mock<IGetTenancyByIdUseCase>();
+            _classUnderTest = new TenancyInformationController(_getByIdMock.Object);
+        }
+
+        [Test]
+        public void ViewRecordReturnsBadRequestForInvalidArguments()
+        {
+            // 1231 is not a valid tag_ref - should return BadRequest400
+            var response = _classUnderTest.ViewRecord("1231") as ObjectResult;
+            response.Should().NotBeNull();
+            response?.StatusCode.Should().Be(400);
```

Comment:
Might be good to also return a message here explaining the reason the 400 is thrown. Not too specific, just something like - Tag Ref isn't in a valid format

Reply:
yeah thats good, and yeah you can just add an assertion to this test